dataclass_function.py

```diff
@@ -92,3 +94,108 @@ class School(AsFrame):
     rrcp_school: Data[str]
     rrcp_area: Data[int]
     rrcp_country: Data[int]
+
+
+# function for getting list from df columns
+
+
+def createdf(file):
+    # read the file
+    df = pd.read_excel(file)
+
+    def clmnlist(i):
+        return list(df.iloc[:, i])
+
+    """ For the school df, showing:
+        school_id
+        rrcp_school
+        rrcp_area
+        rrcp_country
+    """
+    school_df = School.new(clmnlist(6), clmnlist(5), clmnlist(4), clmnlist(3))

```

Comment:
would we need to make the schools unique first?

it may handle it but I imaginged making a new df which is just the school columns, unique and then create all of the schools like this?

You can then remove the school columns for the next step

Reply:
Ah yes that would be much nicer, will update this